trace-dispatcher/doc/trace-dispatcher.md

```diff
@@ -263,6 +263,10 @@ class MetaTrace a where
 
 Metrics are seamlessly incorporated into the system through regular trace messages implementing the `asMetrics` function within the `LogFormatting` typeclass. Unlike other trace components, metrics are not subjected to filtering and are consistently provided. This occurs as long as the `EKGBackend` is configured for the message. The `EKGBackend` then forwards these metrics to `cardano-tracer` for additional processing. Subsequently, they are dispatched as Prometheus metrics, extending their utility and visibility.
 
+It is essential to implement the metricsDoc function of the MetaTrace typeclass, as this information is utilized to optimize system performance.
+
+The configuration option TraceOptionMetricsPrefix can be used to prepend a prefix to any trace message. For example, the prefix could be "cardano.node".
+
```

Comment:
Does the option prepend the prefix to any *trace message*, or just to any *metric name*?